src/components/App.tsx

```diff
@@ -1,16 +1,18 @@
-import { useMoviesContext } from "../stateManagement/store";
 import Body from "./Body/Body";
 import Footer from "./Footer/Footer";
 import Header from "./Header/Header";
 import MovieDetails from "./MovieDetails/MovieDetails";
+import { useAppSelector } from "../redux/hooks";
+import { selectMovieDetailsStatus } from "../redux/movieDetailsSlice";
 
 import styles from "./app.module.css";
+
 function App() {
-  const { movieDetailsIsOpen } = useMoviesContext();
+  const movieDetailsStatus = useAppSelector(selectMovieDetailsStatus);
 
   return (
     <div className={styles.App}>
-      {movieDetailsIsOpen ? <Header /> : <MovieDetails />}
+      {movieDetailsStatus ? <MovieDetails /> : <Header />}
       <Body />
```

Comment:
The user must not see this:
![image](https://user-images.githubusercontent.com/106384184/209787586-4fb4bc88-500d-41c8-bf7b-0d3d2c41e248.png)